API Project/src/main/java/com/bounswe/services/CulturalHeritageService.java

```diff
@@ -35,5 +35,10 @@ public void save(CulturalHeritage culturalHeritage) {
     while(it.hasNext()) culturalHeritages.add(it.next());
     return culturalHeritages;
   }
+
+  public void delete(CulturalHeritage culturalHeritage){
+	this.culturalHeritageRepository.delete(culturalHeritage);
```

Comment:
Oh it's fine then. Can you test this version and see if it works? You can merge when you are sure it works @bounaykut